packages/uploadthing/src/internal/logger.ts

```diff
@@ -50,7 +54,7 @@ function formatStack(stack: string) {
         l
           .trim()
           .replace("file://", "")
-          .replace(process.cwd() + "/", ""),
+          .replace(cwd + "/", ""),
```

Comment:
there is some possibility that there is `unknown/` in the stack, might want to use a more unique value

Reply:
I did think of that but just figured unknown was pretty usique but I guess it isn't... let's add some special characters in there to be sure